app/scripts/modules/core/src/managed/ManagedResourceHistoryModal.tsx

```diff
@@ -166,7 +205,7 @@ export const ManagedResourceHistoryModal = ({ resourceSummary }: IManagedResourc
               </button>
             </div>
           )}
-          {historyEventStatus === 'RESOLVED' && (
+          {!isLoading && historyEventStatus !== 'REJECTED' && (
```

Comment:
all this loading related logic is primarily to ensure that when a polling request happens and the result from `usePollingData` switches to `PENDING`, we don't a loading indicator, but do if we're making a request after one failed.

Reply:
Might be worth just making that another variable so the intent is a bit more clear?

Reply:
👍 very yes. will do.